mod-orders-storage/examples/org_piece_collection.sample

```diff
@@ -0,0 +1,129 @@
+{
+    "orgPieces": [
```

Comment:
we can extend this model with new fields in future, so can rename to more generic name